src/app/domains/Condition/components/ConditionTemplates/PlainShortTextStringTemplate/PlaneShortTextStringTemplate.js

```diff
@@ -0,0 +1,90 @@
+import { Box, Col, Row } from '@qonsoll/react-design'
+import { Button, Input, Select } from 'antd'
+import Text from 'antd/lib/typography/Text'
+import React from 'react'
+import { TEXT_CONDITION_RULES_VALUE } from 'app/constants/planeTextStringConditionRules'
+import { QuestionSelect } from 'domains/Question/components'
+import PropTypes from 'prop-types'
+import { styles } from './PlainShortTextStringTemplate.style'
+import { globalStyles } from 'app/styles'
+
+const { Option } = Select
+
+function PlaneShortTextStringTemplate(props) {
+  const { answers, id, addCondition, questionList, addRedirectQuestion } = props
+  // const [conditionArray, setConditionArray] = useState([''])
+  // [CLEAN FUNCTIONS]
+
+  const onClick = () => {
+    addCondition('')
```

Comment:
check if there no bug

Reply:
maybe you should use smth like that addCondition({ name: '', redirectQuestion: null })